CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [#18531](https://github.com/cosmos/cosmos-sdk/pull/18531) Baseapp's `GetConsensusParams` returns an empty struct instead of panicking if no params are found.
 * (client/tx) [#18472](https://github.com/cosmos/cosmos-sdk/pull/18472) Utilizes the correct Pubkey when simulating a transaction.
 * (baseapp) [#18486](https://github.com/cosmos/cosmos-sdk/pull/18486) Fixed FinalizeBlock calls not being passed to ABCIListeners.
+* (baseapp) [#18627](https://github.com/cosmos/cosmos-sdk/pull/18627) Post handlers are run on non successful transaction executions too.
```

Comment:
I am for merging this into v0.50.2 with the assumption that there are no user of v0.50.1 on prod yet.
With the amount of bug fixes we will add to v0.50.2, I think no one with launch on prod with v0.50.1.
I have checked sourcegraph and [github](https://github.com/search?q=github.com%2Fcosmos%2Fcosmos-sdk+v0.50.1+path%3A**%2Fgo.mod+-org%3Acosmos+-org%3Acosmwasm+-org%3Aignite+-org%3Astrangelove-ventures&type=code) which seems to confirm that. Maybe only berachain (so cc-ing @itsdevbear)?

Reply:
Seems reasonable to me. ACK

Reply:
We have a testnet on v0.50.1 yuh.

Reply:
We don't use post handler so this is non consensus breaking for us. All good